### ISSUE TITLE
[Major] No way to view the final meeting/no information is saved.

### ISSUE THREAD

**Issue by BossJ** [EZPick/EasyPick#37]
1. Go to URL (ezpick.herokuapp.com)

2. Fill out appropriate fields.

3. Click next to move to the next page.

4. Add an email to the invite list and invite.

5. Open email and click respond.

- This is where the error starts; no information about scheduled meeting is saved.
- Email used to respond is not saved; you need to reenter said email.

6. Fill out response form and submit.

7. Reach submission page.

- Still no indication of when my meeting will occur or meeting information is present.
- No email sent to users after submission regarding submission results/no view for submission.

**Comment by zmbc** [comment]
Thanks for the report!

I believe this is a duplicate of #25 (although that's not obvious from the title). The page where you could see the meeting location and who's responded is the meeting page ([here's the meeting page for your meeting](http://ezpick.herokuapp.com/meeting/40)). We simply forgot to give you that link and also sent the invite emails with the wrong link. That will be fixed once #28 is merged.

I'm closing this issue in favor of #25. Feel free to re-open if you think I've missed anything.
